src/controller/controller.cpp

```diff
@@ -34,15 +34,21 @@ long Controller::run() {
 }
 
 bool Controller::simulate() {
-    if (RENDER_TERMINALVIEW) terminalview.render(world);
+    if (RENDER_TERMINALVIEW)
+        terminalview.render(world);
```

Comment:
Please add curly brackets {} here too (or inline it again), that's less confusing than this

Reply:
af03fb5

Reply:
af03fb5